### DISCUSSION TITLE
safetyGraphics v2.0 WG - 2021-01-08

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#471] [Meetings]
# Attendees
- @jwildfire 
- @xni7 
- @rescalan 
- @MayaGans 
- @elimillera 
- @bzkrouse 
- @sibyllwangxxxx

# Agenda/Notes

1. Introducing @bzkrouse 
2. Annual Goal overview
3. Sprint planning / Work assignments [project](https://github.com/SafetyGraphics/safetyGraphics/projects/14)
   - Jeremy Wildfire - safetyData set up / safetyCharts migration / github actions set up
   - Xiao Ni - App Creation Workflow #427 / Close out widget init issue #465
   - Renan Escalante - safetyCharts testing - starting with safetyHistogram static renderer.
   - Maya Gans - Forest Plot chart. Designated safetyGraphics bug squashing duty.  
   - Eli Miller - safetyCharts tplyr example - https://github.com/SafetyGraphics/safetyCharts/pull/38
   - Becca Krouse - Chart Export? #449 
   - Bo - App Creation Workflow #427 
4. "App Builder" PR from @xni7 [branch](https://github.com/SafetyGraphics/safetyGraphics/tree/dev-workflow)
5. safetyData Update [link](https://github.com/SafetyGraphics/safetyData) 

# Action Items

- Everyone will work on assignments above!
- Jeremy to schedule chat with Maya re: forest plot. 
- @xni7 to make PR for workflow widget - Jeremy & Bo to review - others optional.

**Comment by sibyllwangxxxx** [comment]
@sibyllwangxxxx is Bo Wang on GitHub btw :)

**Comment by xni7** [comment]
@jwildfire and @sibyllwangxxxx Just opened a PR (https://github.com/SafetyGraphics/safetyGraphics/pull/472) for the last action item. Other feel free to review and comment. 

Will get to #465 this weekend
